pkg/apis/core/validation/validation.go

```diff
@@ -954,7 +953,7 @@ func validateNFSVolumeSource(nfs *core.NFSVolumeSource, fldPath *field.Path) fie
 	if len(nfs.Path) == 0 {
 		allErrs = append(allErrs, field.Required(fldPath.Child("path"), ""))
 	}
-	if !path.IsAbs(nfs.Path) {
+	if !filepath.IsAbs(nfs.Path) {
```

Comment:
/hold

all of the API validation changes should not use filepath... we don't want API validation to depend on the OS the server is running on